tests/model/test_pixel_to_millimeter.py

```diff
@@ -139,8 +141,43 @@ def test_offset_px_mm_strategy():
 
     d = p.to_dict()
 
+    # Panel.from_dict sets the strategy to ParallaxCorrectedPxMmStrategy rather than
+    # OffsetParallaxCorrectedPxMmStrategy as the offsets aren't currently serialized
     pnew = Panel.from_dict(d)
-    assert pnew == p
+    assert pnew != p
 
     pnew = pickle.loads(pickle.dumps(pnew))
-    assert pnew == p
+    assert pnew != p
+
+
+def test_px_mm_strategy_equality():
+    simple = SimplePxMmStrategy()
+    assert simple == SimplePxMmStrategy()
+
+    parallax_corrected = ParallaxCorrectedPxMmStrategy(1, 1)
+    assert parallax_corrected == ParallaxCorrectedPxMmStrategy(1, 1)
+    assert parallax_corrected != ParallaxCorrectedPxMmStrategy(1, 2)
+    assert parallax_corrected != ParallaxCorrectedPxMmStrategy(2, 1)
+    assert parallax_corrected != simple
+
+    dx = flex.double(flex.grid(10, 10), 1)
+    dy = flex.double(flex.grid(10, 10), 1)
+
+    offset = OffsetPxMmStrategy(dx, dy)
```

Comment:
This makes me think I did not see any changes to `OffsetPxMmStrategy()` above - so that the `==` operators for this _do not_ take `dx, dy` value into consideration?

Reply:
The comparison for `OffsetPxMmStrategy` will compare the strategy name, i.e. assert that they are the same class, however it does not explicitly checkt the `dx, dy` values. I toyed with explicitly comparing the values, but thought that adding boolean comparions of e.g. 16 million pixels might add unnecessary overhead.

Reply:
You're probably not wrong here, however perhaps I would suggest a comment to the effect in the code that this _does not_ compare the offset table?